.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on: [push, pull_request]
+jobs:
+  industrial_ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - {OS_NAME: ubuntu,
+             OS_CODE_NAME: bionic,
+             ROS_DISTRO: melodic,
+             ROSDEP_SKIP_KEYS: nvidia-cuda-dev opencv3,
```

Comment:
You probably needed to ignore `opencv3`, as it's not a valid `rosdep` key. See my comment below.